README.md

```diff
@@ -1,110 +1,162 @@
+detect aws credentials...................................................Passed
+detect private key.......................................................Passed
+AWS CloudFormation Linter................................................Failed
+- hook id: cfn-python-lint
+- exit code: 4
+W3011 Both UpdateReplacePolicy and DeletionPolicy are needed to protect Resources/PublicHostedZone from deletion
+core/deploy/dns-zones/template.yaml:20:3
+Checkov..............................................(no files to check)Skipped
+- hook id: checkov
+```
+
+### Installing the dependencies for pre-commit
+
+To use pre-commit locally you will need to install some dependencies, this can be done in 2 ways:
+
+* [Python pip](https://pypi.org/project/pip/)
+* [Homebrew](https://brew.sh/)
+
+#### Using Python pip
+
+Run the following:
 
 ```
 sudo -H pip3 install checkov pre-commit cfn-lint
 ```
 
-this should work across platforms
+This should work across platforms.
 
-#### Method 2 - Brew
+#### Using Homebrew
 
-If you have brew installed please run the following:
+Run the following:
 
 ```
 brew install pre-commit ;\
 brew install cfn-lint ;\
 brew install checkov
 ```
 
-### Post Installation Configuration
+#### Configuring pre-commit
 
-once installed run:
+Once installed, run:
 
 ```
 pre-commit install
 ```
 
-To update the various versions of the pre-commit plugins, this can be done by running:
+Update the pre-commit plugins by running:
 
 ```
 pre-commit autoupdate && pre-commit install
 ```
 
-This will install / configure the pre-commit git hooks, if it detects an issue while committing it will produce an output like the following:
+This will install and configure the pre-commit Git hooks. 
 
-```
- git commit -a
-check json...........................................(no files to check)Skipped
-fix end of files.........................................................Passed
-trim trailing whitespace.................................................Passed
-detect aws credentials...................................................Passed
-detect private key.......................................................Passed
-AWS CloudFormation Linter................................................Failed
-- hook id: cfn-python-lint
-- exit code: 4
-W3011 Both UpdateReplacePolicy and DeletionPolicy are needed to protect Resources/PublicHostedZone from deletion
-core/deploy/dns-zones/template.yaml:20:3
-Checkov..............................................(no files to check)Skipped
-- hook id: checkov
-```
+## GitHub tools and workflows
+
+### GitHub CODEOWNERS
+
+You can use a [GitHub CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file to define individuals or teams that are responsible for code in a repository. Code owners are automatically requested for review when someone opens a pull request that modifies code that they own.
```

Comment:
```suggestion
You can use a [GitHub CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file to define individuals or teams that are responsible for code in a repository. Github automatically requests reviews from these code owners when someone opens a pull request that modifies the code they own.
```
A suggestion to avoid passive voice.